utils/generate_grammar_tables.py

```diff
@@ -77,6 +77,16 @@ def make_path_to_file(f):
             raise
 
 
+def convert_min_required_version(version):
+    """Converts the minimal required SPIR-V version encoded in the
+    grammar to the symbol in SPIRV-Tools"""
+    if version is None:
+        return 'SPV_ENV_UNIVERSAL_1_0'
+    if version == 'None':
+        return '0xffffffffu'
```

Comment:
I'm not sure about creating `SPV_ENV_UNIVERSAL_infinity` because it will be available to API consumers and we need do sanity check for it. But I don't have strong preferences. I can change if you have.

Reply:
This seems fine.